Dockerfile

```diff
@@ -468,6 +536,20 @@ RUN set -x; \
 	&& cd MathJax \
 	&& git checkout -b $MW_VERSION 4afdc226f08f9c2b1471a523d3c64df716b25c6c
 
+# SimpleMathJax
+RUN set -x; \
+	cd $MW_HOME/extensions \
+	&& git clone https://github.com/jmnote/SimpleMathJax.git \
+	&& cd SimpleMathJax \
+	&& git checkout -b $MW_VERSION ddcac9ac1616aed794576f2914ee426879194f0f
+
+# Wiretap
+RUN set -x; \
+	cd $MW_HOME/extensions \
+	&& git clone https://github.com/enterprisemediawiki/Wiretap.git \
+	&& cd Wiretap \
+	&& git checkout -b $MW_VERSION a97b708c3093ea66e7cf625859b1b38178526bab
```

Comment:
I'd propose to keep adding new extensions (which does not require patches) into the RUN statement above, though, as long as we don't have an auto-update mechanism and unless we're out of layers [1] I am not going to treat this as blocker

1. https://docs.docker.com/storage/storagedriver/overlayfs-driver/#:~:text=The%20overlay2%20driver%20natively%20supports,inodes%20on%20the%20backing%20filesystem.

Reply:
why we added them to one RUN statement? I'd propose to separate all commands inside "source" build because it look nicer and allows to use cache more effectively (and does not increases number of layers)

Reply:
I'd be happy to split it all into separate `RUN`, unfortunately, it still does increase the number of layers during stage build hence we'll quickly hit the 128 limit of the overlayfs. 

The reason for this is that stage still needs to be saved as an intermediate image, so all the common limitations still apply. You can verify this by building the image from scratch and then pick intermediate stages images from `docker image ls` and inspect via `docker history` 

Actually, the first version of the generator script was generating individual `RUN` statements for each extension and was changed to a single `RUN` exactly because it was exceeding the number of layers.

Reply:
I think it is time to remember my idea with rpm packages, I created https://wikiteq.atlassian.net/browse/WIK-320